ngraph/core/src/op/convert.cpp

```diff
@@ -115,6 +116,7 @@ namespace convert
             NGRAPH_TYPE_CASE(evaluate_convert, u64, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, f16, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, f32, arg, out);
+            NGRAPH_TYPE_CASE(evaluate_convert, boolean, arg, out);
```

Comment:
Why do we need to have convert for boolean?

Reply:
In the model I'm working on, there are subgraphs with Convert ops on booleans that can be easily constant folded. It's a small change so I thought I can include it here.

Reply:
Can we add new tests to check that it works as expected?

Reply:
Tests added